src/lib/jsonParseV2/parseAttributeValue.cpp

```diff
@@ -49,7 +49,8 @@ std::string parseAttributeValue(ConnectionInfo* ciP, ContextAttribute* caP)
   if (document.HasParseError())
   {
     alarmMgr.badInput(clientIp, "JSON parse error");
-    oe.fill(SccBadRequest, "Errors found in incoming JSON buffer");
+    oe.reasonPhrase = ERROR_STRING_PARSERROR;
```

Comment:
Doesn't work that way:

```
OrionError::OrionError(HttpStatusCode _code, const std::string& _details)
{
  code          = _code;
  reasonPhrase  = httpStatusCodeString(code);
  details       = _details;
}
```

The reasonPhrase  is automatically derived from SccBadRequest (i.e. something like "Bad Request"), but in this case I want to use a different one. And I don't what details be "ERROR_STRING_PARSERROR", but "Errors found in incoming JSON buffer".